.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
+    "es2015",
```

Comment:
Ah, I see - `import DashboardPlugin from 'webpack-dashboard/plugin';`. Yeah, removing CommonJS module conversion screws up the use of `import`. (Even though, as Sarah pointed out, `import` is the proper way to do it in ES6 and `require` is CommonJS... that's a bit of a WTF)

I'll test to see that we can just run `const DashboardPlugin = require('webpack-dashboard/plugin');` instead, and if so, I'll push a PR shortly. Apologies; I should have caught this in a review before the merge.